main.go

```diff
@@ -90,7 +104,14 @@ func main() {
 	// Create a new Cmd to provide shared dependencies and start components
 	klog.Info("setting up manager")
 	mgr, err := manager.New(workloadConfig, manager.Options{
-		MetricsBindAddress: metricsPort,
+		MetricsBindAddress:         metricsPort,
```

Comment:
Please add `ReleaseOnCancel: true` else this will cause approx 3 minute delays during upgrades